packages/server/graphql/mutations/inviteToTeam.ts

```diff
@@ -57,8 +57,8 @@ export default {
 
       // AUTH
       const viewerId = getUserId(authToken)
-      if (!isTeamMember(authToken, teamId)) {
-        return standardError(new Error('Team not found'), {userId: viewerId})
+      if (!isTeamMember(authToken, teamId) && !isSuperUser(authToken)) {
+        return standardError(new Error(`Viewer does not belong to that team`), {userId: viewerId})
```

Comment:
-1 I think we put "Team not found" here for security reasons. At least we used this consistently everywhere.

Reply:
True, I guess we want to make that message unclear for potential attackers. I can see a few places where we're not doing that, but it seems safer to keep the old error message. Update pushed!